docker/py3-tools-ubi/verify.py

```diff
@@ -0,0 +1 @@
+../py3-tools/verify.py
```

Comment:
is this enough? we dont need to import it?

Reply:
yes its enough, look at the run of the build and see